### DISCUSSION TITLE
Kmonad NonExistentTarget ("documents","docs") (macOS)

### DISCUSSION THREAD

**Discussion by alloc33** [manna-harbour/miryoku#129] [Q&A] [answered]
when I pass .kbd to kmonad executable I got this:
```
kmonad: NonExistentTarget ("documents","docs")
```
Is there any way to fix that? Or may be I do something wrong?
macOS Monterey 12.5 (M1 pro)

**Comment by alloc33** [comment]
found the answer here
https://github.com/kmonad/kmonad/issues/605#issuecomment-1211113178

**Comment by manna-harbour** [comment, marked as answer]
Please use the Miryoku Discussions Q&A category for questions and not the Issues. Miryoku Discussions and Support is described here: https://github.com/manna-harbour/miryoku/discussions/8. You can use this link to ask a question: https://github.com/manna-harbour/miryoku/discussions/new?category=q-a.

Before asking a question on the Miryoku Discussions, first try to determine if the issue is specific to Miryoku or not.  The following guide is for Miryoku ZMK Support, but the same principles apply for Miryoku QMK and Miryoku KMonad, substituting their respective discords and default keymaps. For KMonad you can use the turorial keymap as the default.

> ### Miryoku ZMK Support
> The home for ZMK support is help in the ZMK discord (https://zmk.dev/community/discord/invite). The home for Miryoku support is Miryoku Discussions and Support (https://github.com/manna-harbour/miryoku#discussions-and-support). For Miryoku ZMK support please choose between those as follows.
> 1. If the issue is specific to ZMK but not specific to Miryoku ZMK, choose ZMK support. If you have the same issue with the default keymap for your keyboard then the issue is not specific to Miryoku ZMK. If you have built a new keyboard and it doesn't work as expected, first try building and flashing the default keymap for your keyboard according to the ZMK docs.
> 2. If the issue is specific to Miryoku but not specific to Miryoku ZMK, choose Miryoku support. If the same issue applies to other Miryoku implementations then it's not specific to Miryoku ZMK. If the issue relates to something covered in the Miryoku docs and not the Miryoku ZMK docs it's likely to be specific to Miryoku and not Miryoku ZMK, and vice versa.
> 3. If the issue is specific to Miryoku ZMK choose either ZMK support or Miryoku support as seems most appropriate. If the issue is likely to also be of interest to those not using Miryoku, choose ZMK support and @ mention Manna Harbour in your message. If not, choose Miryoku support.

Given the above, don't ask the same question in a Miryoku Discussion and in a platform discord at the same time. Also, don't ask the same question in multiple discords at the same time.

> when I pass .kbd to kmonad executable I got this:
>
> kmonad: NonExistentTarget ("documents","docs")
> 
> Is there any way to fix that? Or may be I do something wrong? macOS Monterey 12.5 (M1 pro)

Using this as an example, you would try the default keymap for KMonad (the tutorial) and if you get the same error you will know that the issue is not specific to Miryoku KMonad. You would then ask for help on the KMonad discord and not in the Miryoku Discussions.
